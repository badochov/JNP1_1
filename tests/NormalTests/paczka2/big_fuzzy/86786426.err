Error in line 2: OIgUgIW A876 1,6
Error in line 3: WcJjIHMyJx A176 24,0
Error in line 33: WcJjIHMyJx A176 15,0
Error in line 28: A176 S14 62,9
Error in line 35: 1N2MI7Ry A176 27,6
Error in line 38: 10mrPY S825 15,8
Error in line 51: 10mrPY S14 75,0
Error in line 55: Cds S825 17,46
Error in line 52: 10mrPY A876 56,6
Error in line 54: Cds A176 247,8
Error in line 57: Cds A876 34,4
Error in line 67: 10mrPY S825 73,7
Error in line 65: 1N2MI7Ry S825 27,2
Error in line 79: A176 A876 2,3
Error in line 85: Cds S14 59,5
Error in line 87: Cds A176 36,1
Error in line 90: A176 A176 62,7
Error in line 94: OgUgW A876 18,1
Error in line 93: WcJjIHMyJx A176 20,8
Error in line 102: 10mrPY S825 89,2
Error in line 97: OIgUgIW A876 39,1
Error in line 117: 10mrPY A876 64,3
Error in line 122: 1N2MI7Ry A876 10,3
Error in line 129: 1N2MI7Ry S14 89,2
Error in line 136: 10mrPY S825 34,2
Error in line 134: OIgUgIW A176 54,7
Error in line 137: WcJjIHMyJx S14 79,2
Error in line 103: A176 S825 14,2
Error in line 157: WcJjIHMyJx S14 77,5
Error in line 168: Cds S14 62,5
Error in line 166: WcJjIHMyJx S825 13,7
Error in line 171: WcJjIHMyJx A176 42,1
Error in line 176: 10mrPY S825 16,8
Error in line 182: 1N2MI7Ry S825 73,6
Error in line 208: OIgUgIW A876 59,89
Error in line 211: Cds A876 36,4
Error in line 214: Cds S825 75,9
Error in line 224: WcJjIHMyJx Sv4 v8,3
Error in line 229: KJ3 S14 24,5
Error in line 236: KJ3 S825 49,8
Error in line 242: @N2MI7Ry S@4 7@,4
Error in line 243: OIgUgIW A176 -4,4
Error in line 240: Cds A876 75,2
Error in line 237: WcJjIHMyJx S825 201,9
