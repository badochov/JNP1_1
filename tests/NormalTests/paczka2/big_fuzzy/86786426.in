?S14
OIgUgIW A876 1,6
WcJjIHMyJx A176 24,0
Cds A176 64,5
Cds A176 134,8
?OIgUgIW
OIgUgIW S14 74,8
10mrPY S825 96,6
A176 S14 2,7
A176 S14 10,3
1N2MI7Ry A176 78,7
10mrPY S825 368,0
KJ3 A176 51,3
OIgUgIW S14 228,9
WcJjIHMyJx S825 38,7
WcJjIHMyJx S825 10,4
KJ3 A176 41,6
A176 A876 53,7
1N2MI7Ry A176 88,9
A176 A876 73,2
10mrPY S14 28,3
10mrPY S14 67,4
WcJjIHMyJx A176 91,6
1N2MI7Ry A876 96,7
?A176

10mrPY S825 70,7
A176 S14 62,9
1N2MI7Ry A876 416,8
WcJjIHMyJx A176 4,2
Cds S825 2,1
Cds S825 4,6
WcJjIHMyJx A176 15,0
WcJjIMyJx A876 99,8
1N2MI7Ry A176 27,6
10mrPY S825 6,8
WcJjIHMyJx A876 52,3
10mrPY S825 15,8
Cds A176 89,2
A176 A176 50,1
A176 A176 57,0
A176 A176 12,5
WcJjIHMyJx A876 202,4
1N2MI7Ry S825 57,3
1N2MI7Ry S825 240,7
?

OIgUgIW S14 84,4

Cds A176 63,7
10mrPY S14 75,0
10mrPY A876 56,6

Cds A176 247,8
Cds S825 17,46
10mrPY A176 90,5
Cds A876 34,4
OIgUgIW S14 165,4
10mrPY A176 427,2
Cds S14 7,9
A176 A176 17,1
Cds S14 38,4
KJ3 S825 7,2
A176 A176 55,9
1N2MI7Ry S825 27,2
WcJjIHMyJx A176 82,9
10mrPY S825 73,7
WcJjIHMyJx A176 378,9
Cds A876 66,9
10mrPY S14 62,4
10mrPY S14 185,3
?WcJjIHMyJx
OIgUgIW A176 41,9
1N2MI7Ry S14 78,5
OIgUgIW A176 32,7
A176 A176 10,1
A176 S14 10,8
A176 S14 19,9
A176 A876 2,3
A176 A176 24,4
A176 A176 7,1
WcJjIHMyJx A876 18,5
Cds A876 143,8
KJ3 S825 11,7
Cds S14 59,5
?10mrPY
Cds A176 36,1
Cds S825 30,5
Cds S825 46,1
A176 A176 62,7
A176 S825 59,1
WcJjIHMyJx A876 62,2
WcJjIHMyJx A176 20,8
OgUgW A876 18,1
WcJjIHMyJx S14 53,0
WcJjIHMyJx S14 174,9
OIgUgIW A876 39,1
A176 S825 5,3

Cds A876 63,1

10mrPY S825 89,2
A176 S825 14,2

10mrPY S14 5,4
10mrPY S14 17,3
OIgUgIW S825 79,1
?
Cds A876 6,6
Cds S14 2,0
WcJjIHMyJx S825 48,1
Cds S14 9,8
?1N2MI7Ry
1N2MI7Ry S14 193,1
WcJjIHMyJx S825 9,6
?
10mrPY A876 64,3

OIgUgIW S825 4,0
10mrPY S825 40,1
OIgUgIW A176 15,7
1N2MI7Ry A876 10,3
1N2MI7Ry S14 23,5
?Cds
1N2MI7Ry S14 72,8
10mrPY S825 8,5
WcJjIHMyJx S14 4,4
OIgUgIW A176 31,8
1N2MI7Ry S14 89,2
1N2MI7Ry S825 0,7
?S14
WcJjIHMyJx S14 10,5
KJ3 S825 18,7
OIgUgIW A176 54,7
?A176
10mrPY S825 34,2
WcJjIHMyJx S14 79,2
KJ3 S825 11,4
Cds S825 7,3
?Cds
?S825
Cds S825 32,5
Cds A176 78,9
10mrPY A176 97,4
10mrPY A176 350,6
1N2MI7Ry S825 1,5
Cds A176 84,4
Cds A176 15,6
KJ3 A176 41,2
OIgUgIW S825 85,0
OIgUgIW S825 381,6
Cds A176 65,1
OIgUgIW A176 16,2
OIgUgIW A176 65,4
WcJjIHMyJx S825 27,9
WcJjIHMyJx S825 12,0
WcJjIHMyJx S14 77,5
?
1N2MI7Ry S14 3,7
?A876
A176 A876 16,8
OIgUgIW A876 92,9
OIgUgIW A876 360,5
A176 A876 24,9
1N2MI7Ry S14 12,9
WcJjIHMyJx S825 13,7
?A176
Cds S14 62,5
OIgUgIW A876 35,1
Cds A176 13,6
WcJjIHMyJx A176 42,1
OIgUgIW A876 123,6
Cds A176 49,5
KJ3 A176 56,9
A176 A176 10,9
10mrPY S825 16,8
A176 A176 7,0
WcJjIHMyJx A876 47,0
10mrPY A876 8,4
Cds S14 96,0
?Cds
1N2MI7Ry S825 73,6


?10mrPY
OIgUgIW A176 41,7

OIgUgIW A176 44,6
OIgUgIW A176 28,0
1N2MI7Ry A876 22,9
1N2MI7Ry A876 109,7
10mrPY A876 71,8
?A876
OIgUgIW A176 5,9
WcJjIHMyJx A876 63,9
WcJjIHMyJx A176 16,3
WcJjIHMyJx A176 9,9
1N2MI7Ry S14 2,6
?
?S825

1N2MI7Ry S14 12,5
?10mrPY
?
KJ3 S825 12,7
KJ3 S825 22,1
10mrPY S825 74,7
OIgUgIW A876 59,89
Cds S14 418,6
OIgUgIW A876 11,8
Cds A876 36,4
?WcJjIHMyJx
OIgUgIW A876 46,5
Cds S825 75,9
1N2MI7Ry S14 51,4
1N2MI7Ry S14 230,8
Cds A876 8,3
WcJjIHMyJx S14 2,7
WcJjIHMyJx S14 8,1
?
1N2MI7Ry S825 98,3
KJ3 S825 9,1
KJ3 S825 25,6
WcJjIHMyJx Sv4 v8,3
WcJjIHMyJx S14 90,8
1N2MI7Ry S825 373,5
10mrPY S825 8,5
10mrPY S825 40,3
KJ3 S14 24,5
Cds A876 20,3
WcJjIHMyJx S14 73,1
WcJjIHMyJx S825 23,3
OIgUgIW A176 32,1
?
WcJjIHMyJx S825 62,7
KJ3 S825 49,8
WcJjIHMyJx S825 201,9

OIgUgIW A176 91,1
Cds A876 75,2
KJ3 A876 35,4
@N2MI7Ry S@4 7@,4
OIgUgIW A176 -4,4
Cds S14 14,0
10mrPY S825 26,6
KJ3 A876 21,6
WcJjIHMyJx A176 11,1
1N2MI7Ry S14 14,3
Cds S14 32,5
10mrPY S825 108,8