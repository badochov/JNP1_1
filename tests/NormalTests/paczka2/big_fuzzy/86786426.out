A176 A 19,5 S 7,6
A176 90,2
10mrPY S 374,4
1N2MI7Ry A 330,3 S 183,4
A176 A 26,4 S 7,6
Cds A 70,3 S 2,5
KJ3 A 9,7
OIgUgIW S 154,1
WcJjIHMyJx A 237,5 S 28,3
S14 200,8
A176 184,5
S825 549,5
A876 489,7
WcJjIHMyJx A 533,5 S 28,3
10mrPY A 336,7 S 497,3
10mrPY A 336,7 S 509,2
1N2MI7Ry A 330,3 S 183,4
A176 A 94,1 S 70,5
Cds A 172,7 S 48,6
KJ3 A 9,7 S 4,5
OIgUgIW A 9,2 S 235,1
WcJjIHMyJx A 577,2 S 150,2
S14 578,1
A176 919,6
S825 623,4
A876 610,3
1N2MI7Ry A 330,3 S 183,4
10mrPY A 336,7 S 509,2
1N2MI7Ry A 330,3 S 298,0
A176 A 94,1 S 70,5
Cds A 229,2 S 56,4
KJ3 A 9,7 S 4,5
OIgUgIW A 9,2 S 235,1
WcJjIHMyJx A 577,2 S 188,7
S14 700,5
A176 919,6
S825 661,9
A876 666,8
Cds A 229,2 S 56,4
S14 749,8
A176 A 94,1 S 70,5
A176 935,7
Cds A 229,2 S 56,4
S825 775,9
10mrPY A 589,9 S 540,8
1N2MI7Ry A 330,3 S 348,1
A176 A 94,1 S 70,5
Cds A 284,2 S 81,6
KJ3 A 9,7 S 11,8
OIgUgIW A 74,5 S 606,8
WcJjIHMyJx A 577,2 S 210,7
S14 755,9
A176 1293,1
S825 1114,4
A876 666,8
A876 666,8
A176 A 102,2 S 70,5
A176 1293,1
Cds A 320,1 S 81,6
10mrPY A 589,9 S 540,8
A876 1181,2
10mrPY A 653,3 S 540,8
1N2MI7Ry A 417,1 S 357,3
A176 A 106,1 S 70,5
Cds A 320,1 S 81,6
KJ3 A 25,4 S 11,8
OIgUgIW A 455,6 S 606,8
WcJjIHMyJx A 600,5 S 210,7
S14 765,1
A176 1380,0
S825 1114,4
A876 1198,1
S825 1114,4
10mrPY A 653,3 S 540,8
10mrPY A 653,3 S 540,8
1N2MI7Ry A 417,1 S 367,2
A176 A 106,1 S 70,5
Cds A 320,1 S 81,6
KJ3 A 25,4 S 11,8
OIgUgIW A 455,6 S 606,8
WcJjIHMyJx A 600,5 S 210,7
S14 775,0
A176 1380,0
S825 1114,4
A876 1198,1
WcJjIHMyJx A 600,5 S 210,7
10mrPY A 653,3 S 540,8
1N2MI7Ry A 417,1 S 546,6
A176 A 106,1 S 70,5
Cds A 320,1 S 404,2
KJ3 A 25,4 S 21,2
OIgUgIW A 490,3 S 606,8
WcJjIHMyJx A 600,5 S 216,1
S14 1282,4
A176 1380,0
S825 1123,8
A876 1232,8
10mrPY A 653,3 S 607,0
1N2MI7Ry A 417,1 S 821,8
A176 A 106,1 S 70,5
Cds A 332,1 S 404,2
KJ3 A 25,4 S 37,7
OIgUgIW A 490,3 S 606,8
WcJjIHMyJx A 600,5 S 233,8
S14 1300,1
A176 1380,0
S825 1481,7
A876 1244,8
